File: 7KYU/Jumping Number (Special Numbers Series #4)/solution.cs
using System;
class Kata
{
    public static string JumpingNumber(int number)
    {
      Console.WriteLine(number);
       var split = number.ToString().ToCharArray();
       for(var i = 0; i < split.Length -1; i++)
         if(Math.Abs(split[i] - split[i+1]) > 1)
            return "Not!!";
        return "Jumping!!";
    }
}

File: 7KYU/Jumping Number (Special Numbers Series #4)/solution.clj
(ns kata)

(defn number->digits [num]
  (->> num str (map (comp read-string str))))

(defn jumping-number [n]
  (let [digits (number->digits n)
        jumping? (->> (map (fn [x y] (= 1 (Math/abs (- x y)))) digits (rest digits))
                      (every? true?))]
    (if jumping?
      "Jumping!!"
      "Not!!")))


